main.go

```diff
@@ -114,12 +152,14 @@ func main() {
 
 						lag := offset - partitionOffsets[partitionID]
 
-						if *useTags {
-							var tags []string
-							tags = append(tags, "topic="+topic)
-							tags = append(tags, fmt.Sprintf("partition=%d", partitionID))
-							tags = append(tags, "consumer_group="+cg)
-							stats.Gauge(fmt.Sprintf("consumer_lag,%s", strings.Join(tags, ",")), lag)
+						stats := statsdClient.Clone(
```

Comment:
Does Go garbage collect automatically?

Reply:
Yes, go is a garbage collected language.  The cloned client also doesn't need to be `.Close()`d because when the main client is closed, all cloned clients are also closed.